samples/basic/authzhelper.go

```diff
@@ -62,19 +62,14 @@ func provisionObject(ctx context.Context, authZClient *authz.Client, typeID uuid
 	return obj.ID, nil
 }
 
-func provisionUser(ctx context.Context, idpClient *idp.Client, username, password string, profile idp.UserProfile) (uuid.UUID, error) {
-	var err error
-	if _, err = idpClient.CreateUserWithPassword(ctx, username, password, profile); !isBenign(err) {
-		return uuid.Nil, ucerr.Wrap(err)
-	}
-	users, err := idpClient.ListUsersForEmail(ctx, profile.Email, idp.AuthnTypePassword)
-	if err != nil {
-		return uuid.Nil, ucerr.Wrap(err)
-	}
-	if len(users) != 1 {
-		return uuid.Nil, ucerr.Errorf("expected 1 user with email '%s', got %d", profile.Email, len(users))
+func provisionUser(ctx context.Context, idpClient *idp.Client, name string) (uuid.UUID, error) {
+	// Try to find user by existing alias
```

Comment:
@g-woss-gh is this the recommended way to use external alias to ensure we don't create new users every time this sample is run?

Reply:
I believe so; this was a change that @stgarrity had worked on as part of the userstore sample work for WhatNot before the holidays, so he can probably confirm.

Reply:
Yep, lgtm. Not needed unless you want to, but seems like a good use (that doesn't require hard coding a known uuid)